networking_dvs/utils/spec_builder.py

```diff
@@ -23,13 +23,16 @@ def __init__(self, spec_factory):
     def pg_config(self, default_port_config):
         spec = self.factory.create('ns0:DVPortgroupConfigSpec')
         spec.defaultPortConfig = default_port_config
-        policy = self.factory.create('ns0:DVPortgroupPolicy')
+        policy = self.factory.create('ns0:VMwareDVSPortgroupPolicy')
         policy.blockOverrideAllowed = '1'
         policy.livePortMovingAllowed = '0'
         policy.portConfigResetAtDisconnect = '1'
         policy.shapingOverrideAllowed = '0'
         policy.trafficFilterOverrideAllowed = '1'
```

Comment:
[observed during tests on staging] no longer necessary to allow traffic filtering overrides on port level